NossoMercadoLivre/Controllers/UserController.cs

```diff
@@ -9,7 +9,7 @@ namespace NossoMercadoLivre.Controllers
     [Route("api/User")]
     [Produces("application/json")]
     [ApiController]
-    public class UserController : Controller
+    public class UserController : Base
```

Comment:
Pq todo mundo tem que herdar aqui?

Reply:
Essa classe Base é onde eu centralizo tudo que for comum entre os Controllers, dai eu centralizei o GetUserId, que é onde eu pego o Id do cara que ta logado e faço qualquer lógica dentro dos controllers.

Reply:
Respondendo o comentário geral, essa implementação de JWT é meio que padrão pra ASP.NET Core, e a integração com o Swagger também. Utilizo uma variação dessa implementação no Mobills, mas a gente tem umas camadas a mais de segurança, e criptografia diferente também. Porque não usa mais ? A plataforma ta top, hehe

Reply:
Sobre a classe Base, eu entendi a explicação. Por outro lado não tem nenhum lugar que chama o método de recuperação do usuário ainda. Sugiro apagar :). E quando chegar essa hora, debatemos o uso da herança.